gradle/run.gradle

```diff
@@ -43,9 +43,17 @@ testClusters {
       installedPlugins = Eval.me(installedPlugins)
       for (String p : installedPlugins) {
         plugin('plugins:'.concat(p))
+        if (p.equals("arrow-flight-rpc")) {
```

Comment:
not critical - thinking we maybe could provide a way for plugins to bring their own runtask config if installed.

Reply:
hmm! let me see if it can be done safely as there could be conflicts if exposed via plugin